apps/web/src/components/FeatureFlag.tsx

```diff
@@ -213,7 +213,10 @@ export function FeatureFlag({
             title={currentFlagValue}
             className="max-w-[60px] overflow-hidden text-ellipsis whitespace-nowrap rounded-md bg-gray-600 p-1"
           >
-            {currentFlagValue}
+            {typeof currentFlagValue === "string" &&
```

Comment:
i am no react expert.. but adding "so much" code here seems odd? cant we just have a variable like "trimmedStringValue" or so? to easily acces it here? or even better, "isValueEmptyString"

Reply:
I don't really think it's odd to have it here. The check is only done once and therefore I rather not have a variable which is in-fact only used once. I think this comes down to personal preference